vassal-doc/src/main/readme-referencemanual/ReferenceManual/GlobalOptions.adoc

```diff
@@ -41,6 +41,9 @@ NOTE: If you later need to do arithimetic on this value in a Beanshell expressio
 [#purgeblanks]
 *Purge blank lines from dynamic/global property prompts?*:: When a <<DynamicProperty.adoc#top,Dynamic Property>> or <<SetGlobalProperty.adoc#top,Set Global Property>> trait is configured to prompt users from a list of options, if some of those options are determined by expressions which evaluate to be a blank string, then those options are omitted from the prompt if this option is selected.
 
+[#disableindex]
+*Disable use of location indexes to speed up Global Key Command?*:: A new feature was added to Vassal 3.7 to speed up the execution of all varieties of Global Key Commands (GKC's) where a <<FastMatch.adoc#top,Fast-Match>> 'by location' has been specified. It all also speeds up all Ranged GKCs. The use of these indexes can cause the GKCs to be applied to matching pieces in a different order than previously. Modules should not make any assumptions about the order that GKC's are applied to matching pieces. This option turns off the use of location indexes to speed up GKC's and reverts to the slower pre 3.7 behaviour.
```

Comment:
"It all also" should just be "It also", I believe.
There's also an inconsistency with using an apostrophe when talking about multiple Global Key Commands (throughout the documentation, not just this line); sometimes "GKC's", sometimes "GKCs".  They really should all be "GKCs", IMHO (since the apostrophe signifies ownership, not plurality).

Reply:
Fixed in all doco